index.html

```diff
@@ -0,0 +1,357 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Portfoloio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="header">
+    <div class="Left_side">
+      <a href="#">
+        <div class="logo">
+            My logo
+        </div>
+    </a>
+    </div>
+    <div class="Right_side">
+       <nav>
+        <div class="bar">
+            <i class="fa fa-bars" aria-hidden="true"></i>
+        </div>
+       </nav>
+    </div>
+</header>
+<section class="container">
+  <div class="me">
+    <h1>I’m Mahram Hossain.</h1>
+    <span>Glad to see you!</span>
+    </div>
+        <div class="bio">
+         <p>I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+         </div>
+        <div class="lets_contact">
+          <div class="contact">
+            <h2>LET’S CONNECT</h2>
+        </div>
+       
+            <section class="social_media">
+                <ul class="social-icons-set">
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-twitter" target="_blank" aria-label="Twitter"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-linkedin-in" target="_blank" aria-label="Linkedin"></i></a>
+                  </li>
+                  <li class="social-icon"><a href="#"><i class="fa-solid fa-m" target="_blank"
+                        aria-label="Medium"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-github" target="_blank" aria-label="GitHub"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-angellist" target="_blank" aria-label="Angellist"></i></a>
+                  </li>
+                </ul>
+              </section>
+            </section>
+          </section>
+    </div>
+
+    <!-- Start first card -->
+
+    <section class="middle_section">
+        <div class="pic">
+            <img src="Snapshoot Portfolio.png" alt="">
+        </div>
+        <div class="name">
+            <h1>
+                Mahram Hossain
```

Comment:
- [ ] Great job adding the rest of the work cards but as per the project requirements, you need to ensure that all the cards are enclosed in a `section` tag which will serve as the main container of the projects and should be styled with grid :+1: 
- [ ] I will also suggest that you use `div` tags for each card and not section tags :+1: 
 eg.
```
<section class=works-section>
<div>Card 1</div>
<div>Card 2</div>
<div>Card 3</div>
<div>Card 4</div>
</section>
```
- [ ] Lastly, you can see that from the design there is some spacing between each card kindly add this as well :100: 
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/227758365-bf9fa493-232c-4f7d-ac14-536bb971196b.png)

- Your Design
![image](https://user-images.githubusercontent.com/102699958/227758387-3498093e-6634-441f-9af4-8c8ab93013ec.png)